{{cookiecutter.repo_name}}/compose/prod/django/Dockerfile

```diff
@@ -37,7 +37,26 @@ WORKDIR app
 
 # Install application into container
 COPY . .
-COPY --from=js-deps /{{cookiecutter.repo_name}}/static ./{{cookiecutter.repo_name}}/static
-
-RUN DJANGO_SECRET_KEY="!!! CHANGE ME !!!" DJANGO_ALLOWED_HOSTS="*" APP_VERSION_RELEASE="build" ENVIRONMENT="build" SENTRY_DSN="" REDIS_URL="" DATABASE_URL="" ALLOWED_HOSTS="*" SECRET_KEY="foobar" python ./manage.py collectstatic --noinput
-CMD gunicorn {{cookiecutter.repo_name}}.config.asgi:application --bind 0.0.0.0:8080  -k uvicorn.workers.UvicornWorker --access-logfile - --preload
+COPY --from=js-deps /{{ cookiecutter.repo_name }}/static_source ./{{ cookiecutter.repo_name }}/static_source
+
+ENV DJANGO_SETTINGS_MODULE="{{ cookiecutter.repo_name }}.config.settings.prod"
+RUN DJANGO_AWS_ACCESS_KEY_ID="" \
+    DJANGO_AWS_SECRET_ACCESS_KEY="" \
+    DJANGO_AWS_STORAGE_BUCKET_NAME="" \
+    DJANGO_AWS_S3_ENDPOINT_URL="" \
+    DJANGO_SECURE_SSL_REDIRECT=False \
+    DJANGO_SECURE_HSTS_SECONDS=0 \
+    DJANGO_SECRET_KEY="!!! CHANGE ME !!!" \
+    DJANGO_ALLOWED_HOSTS="*" \
+    APP_VERSION_RELEASE="build" \
+    ENVIRONMENT="build" \
+    SENTRY_DSN="" \
+    REDIS_HOST="" \
+    REDIS_PORT="" \
+    DATABASE_URL="" \
+    ALLOWED_HOSTS="*" \
+    python ./manage.py collectstatic --noinput
+
+# TODO: Set a sensible initial value for workers and threads based on available CPU cores.
+# N.b. Gunicorn timeout must be shorter than any upstream timeouts, e.g. the gateway timeout.
+CMD gunicorn {{ cookiecutter.repo_name }}.config.asgi:application --bind 0.0.0.0:8080 --workers 3 --threads 1 --max-requests 500 --max-requests-jitter 50  --timeout 25  -k uvicorn.workers.UvicornWorker --access-logfile - --preload
```

Comment:
we recently merged in a patch to use a gunicorn config file - can you double check this against the config file?